verifiable-api/types/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+use std::collections::HashMap;
+
+use alloy::{
+    consensus::Account as TrieAccount,
+    eips::BlockId,
+    primitives::{Address, Bytes, B256},
+    rpc::types::{EIP1186StorageProof, Log},
+};
+use serde::{Deserialize, Serialize};
+
+use helios_common::network_spec::NetworkSpec;
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AccountResponse {
+    pub account: TrieAccount,
+    pub code: Bytes,
+    pub account_proof: Vec<Bytes>,
```

Comment:
Why is the account proof just bytes instead of a more structured type like we see in the rpc responses?

Reply:
Not necessarily against this if it is to ensure the response is small. But this brings the next question of why we aren't doing this for storage proofs.

Reply:
The `account_proof` is `Vec<Bytes>` in [Alloy](https://github.com/alloy-rs/alloy/blob/79964358d42ff4f27d3cb21570dd8026c01d1e75/crates/rpc-types-eth/src/account.rs#L60-L61) as well (for the response of `eth_getProof`).

So it is the same.

Reply:
Oh my bad I thought it was just bytes and not a vec of bytes which confused me. This makes sense.